zork.js

```diff
@@ -0,0 +1,329 @@
+//globals
+const readline = require('readline');
+const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+    return new Promise((resolve, reject) => {
+        readlineInterface.question(questionText, resolve);
+    });
+}
+
+class Player{
+    constructor(name,title){
+        this.name = name,
+        this.title = title
+    }
+    inventory = [];
+    drop = function(item){
+        delete this.inventory[item];
+    }
+    take = function(item){
+        this.inventory.push(item);
+    }
+    speak = function(words){
+        console.log(words);
+    }
+    showInventory = function(){
+        console.log(inventory);
+    }
```

Comment:
For these methods either define them using the method syntax you used below, or put them inside the constructor and append the `this` keyword to them.